README.md

```diff
@@ -92,6 +94,9 @@ Some notes:
 - Use option `--java-opts="JAVA_OPTS"` to override the default JAVA_OPTS for the Tomcat process. That's tipically used to set the maximum/initial Heap Memory size (for example: `--java-opts="-Xmx3500m -Xms2500m"`)
 - Use option `--postgis-version=13-3.1-alpine` to specify the PostGIS version to use. By default, 10-2.5-alpine is used.
 - Use option `--debug-port=PORT` to specify the debug port of the Tomcat process.
+- Use option `--glowroot-port=PORT` to specify the APM glowroot port of the Tomcat process.
+- Use option `--glowroot` to use the latest version of glowroot in the Tomcat process.
+- Use option `--glowroot-zip=FILE` to specify the zip file with a version of glowroot to run in the Tomcat process.
```

Comment:
A proposal: instead of adding the 3 options at this section of the readme, create a new one dedicated to glorwoot. Then we can explain what's glowroot, what's an APM, and how is it useful. And then, show the d2-docker start with the options. What do you think?